sys/cccl

```diff
@@ -261,9 +265,9 @@ EOF
         ;;
 
     /*)
-        # All '/' options are assumed to be for cl and are passed through
-        val=$(echo "$1" | cut -c2-)
-        clopt="${clopt},${slash}${val}"
+        # All '/' options are assumed to be absolute path, not options
+        path=$(cygpath -aw "$1")
```

Comment:
Not sure if this is a portable option... E.g. pure mingw installation wont have this